src/SablierV2LockupTranched.sol

```diff
@@ -181,9 +171,17 @@ contract SablierV2LockupTranched is
     //////////////////////////////////////////////////////////////////////////*/
 
     /// @inheritdoc SablierV2Lockup
+    /// @dev The streaming function is:
+    ///
+    /// $$
+    /// f(x) = \Sigma(esa)
+    /// $$
+    ///
+    /// Where:
+    ///
+    /// - $\Sigma(esa)$ is the sum of all elapsed tranches' amounts.
```

Comment:
I think we should call it `eta` - elapsed tranches' amounts.

Reply:
correct